### DISCUSSION TITLE
请问分区数量有没有限制，分区数太多对查询性能会有影响吗

### DISCUSSION THREAD

**Discussion by yaoyi** [apache/doris#12201] [RFC]
之前是按月分区，但考虑目前情况是当天的数据更新比较频繁，所以想尝试按天分区，然后使用分区缓存
但是又担心如果按天分区，3年数据有1000多个分区，不知道对查询性能是否有影响

**Comment by dataalive** [comment]
1000多分区还行，但是你说更新频繁又用分区缓存这是想解决什么问题呢？

**Reply by yaoyi** [reply]
就是文档里描述分区缓存的情况哈，当天的数据会更新比较频繁，客户经常会查询过去7天的数据，这样前几天的数据，应该会利用到分区缓存，提高查询性能，我理解的对吗？
